pages/index.jsx

```diff
@@ -0,0 +1,95 @@
+import { Box, Container, TextField } from "@mui/material";
+import InfoAdd from "../components/InfoAdd"
+import FoodEntryList from "../components/foodEntryList";
+import { PrismaClient } from "@prisma/client";
+import { ObjectSchema } from "yup";
+import axios from "axios";
+import useSWR, { useSWRConfig } from 'swr'
+import { useEffect, useState } from "react";
+
+const fetcher = (url) => axios.get(url).then((response) => response.data);
+
+// const prisma = new PrismaClient();
+
+
+// export async function getServerSideProps() {
+//   console.log(" ********************************** ");
+//   const foodEntries = await prisma.FoodEntry.findMany();
+//   console.log("__________ success _____________");
+//   console.log(foodEntries);
+//   return {
+//     props: {
+//       foodEntries: foodEntries.map((data) => ({
+//         id: data.id,
+//         name: data.name,
+//         calorieValue: data.calorieValue,
+//         takenAt: data.takenAt,
+//       })),
+//     },
+//   };
+// }
+
+
+
+
+
+
+export default function Home() {
+  const { mutate } = useSWRConfig()
+  const {data:foodEntries,error } = useSWR('/api/entryList',fetcher);
+
+  const saveFoodEntry = async (data) =>{
+  try{
+    await fetch("/api/foodEntry",{
+      method: "POST",
+      body: JSON.stringify(data),
+    });
+    const {data:secondMutate,error} = mutate('/api/entryList');
```

Comment:
After using bound mutate, just call `mutate()` here. something like this:
`const data = await mutate();` or just `await mutate();`